src/stores/quick-strategy-store.ts

```diff
@@ -205,6 +207,7 @@ export default class QuickStrategyStore implements IQuickStrategyStore {
         }
 
         this.setFormVisibility(false);
+        botNotification(notification_message().success_quick_strategy_message);
```

Comment:
Lets do it like this instead @mayuran-deriv 

botNotification(notification_message()[NOTIFICATION_TYPE.BOT_IMPORT]);